examples/running-renderer-aws-lambda/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@loki/example-running-renderer-aws-lambda",
+  "version": "0.33.1",
+  "private": true,
+  "scripts": {
+    "build": "sh build.sh",
+    "start": "docker run -d -p 8080:8080 -t example-renderer-aws-lambda:latest"
+  },
+  "dependencies": {
+    "@loki/renderer-aws-lambda": "file:../../packages/renderer-aws-lambda",
```

Comment:
Instead of doing this, add this project to the workspace by adding it to `workspaces.packages` in the root `package.json`

Reply:
I did this way to be able to iterate during the development and have the dependencies updated inside node_modules when I installed them. If I use the version when I install this it will fetch from the npm and not from the code I changed.

Is there another way to achieve this?

I added in the other comment why I didn't use the workspace.

Reply:
There's an option for yarn workspaces called `nohoist` which I think should solve this problem: https://classic.yarnpkg.com/blog/2018/02/15/nohoist/

Reply:
Thanks for the suggestion. I did the change using `workspace` and `nohoist`.